docs/data-sources/cbr_policies.md

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+---
+
+# huaweicloud_cbr_policies
+
+Use this data source to get available CBR policies within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable policy_id {}
+
+data "huaweicloud_cbr_policies" "test" {
+  policy_id = var.policy_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the policies.
+  If omitted, the provider-level region will be used.
+
+* `policy_id` - (Optional, String) Specifies the policy ID used to query.
+
+* `name` - (Optional, String) Specifies the policy name used to query.
+
+* `type` - (Optional, String) Specifies the policy type used to query. The valid values are as follows:
+  + **backup**: Backup policy
+  + **replication**: Replication policy
+
+* `enabled` - (Optional, Bool) Specifies the policy enabling status to query. The valid values are as follows:
+  + **true**: Policy enabled
+  + **false**: Policy not enabled
+
+* `vault_id` - (Optional, String) Specifies the vault ID of the associated policy used to query.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `policies` - All CBR policies that match the filter parameters.
+  The [policies](#CBR_Policies) structure is documented below.
+
+<a name="CBR_Policies"></a>
+The `policies` block supports:
+
+* `id` - The policy ID.
+
+* `name` - The policy name.
+
+* `type` - The protection type of the policy. The valid values are as follows:
+  + **backup**: Backup policy
+  + **replication**: Replication policy
+
+* `enabled` - Whether to enable the policy. The valid values are as follows:
+  + **true**: Policy enabled
+  + **false**: Policy not enabled
+
+* `backup_cycle` - The scheduling rule for the policy backup execution.
+  The [backup_cycle](#CBR_Policies_BackupCycle) structure is documented below.
+
+* `destination_region` - The name of the replication destination region.
+
+* `enable_acceleration` - Whether to enable the acceleration function to shorten the replication time for cross-region.
+  The valid values are as follows:
+  + **true**: Enabled acceleration
+  + **false**: Not enabled acceleration
+
+-> Please note that due to the missing `enable_acceleration` attribute in the API response,
+  so the default value is **false**.
+
+* `destination_project_id` - The ID of the replication destination project.
+
+* `backup_quantity` - The maximum number of retained backups. The value ranges from `2` to `99,999`.
+  This parameter and `time_period` are alternative.
+
+* `time_period` - The duration (in days) for retained backups. The value ranges from `2` to `99,999`.
+
+-> **NOTE:** If `backup_quantity` and `time_period` are both left blank, the backups will be retained permanently.
```

Comment:
I think all notes can be deleted as they are only attributes.

Reply:
fixed.